### DISCUSSION TITLE
Allow categorizing of bookmarks

### DISCUSSION THREAD

**Discussion by upieez** [dailydotdev/daily#913] [Feature Requests]
Currently, when I save an article, it goes into one big bucket aptly named `Bookmarks`. 

Would be really cool if I could create a different list of bookmarks depending on the article I've read. 

Another alternative would be to allow users to filter their bookmarks based on topics because it can be quite bothersome to search through my bookmarks if I have 200 different articles

**Comment by anujbarochia** [comment]
Yes, something like instagram, in which when you save a post you get options to save in particular folder/category

**Comment by idoshamun** [comment]
Hi @upieez,

Would you expect the categorization to happen automatically or some sort of folder mechanism that the user can choose to organize however they want?

**Reply by upieez** [reply]
I would prefer if I could create my own folder mechanism so that I can organize it however I want

**Reply by anujbarochia** [reply]
Yes, folder mechanism is good

**Reply by fpetrovski** [reply]
I was about to request such a feature before finding this discussion is already opened. 
Being able to sort and organize my bookmarks in a folder or list like structure would greatly improve the usefulness of the bookmarks feature.

Maybe have the options to sort by Date Added (Bookmarked) and/or Date Written (The date of the article itself)

**Reply by pstkweb** [reply]
:+1: for the folder like mechanism, I'd like to have this feature to save some articles for future read. Currently all are going to my bookmark list and I use the read/not read style to known what to do with them.
